### PR TITLE
Fix up wording of messages

### PR DESCRIPTION
**Pull request by Volkor3-16**
Hi vorotamoroz, thanks for making this plugin!

I reworded a bunch of messages and UI elements, they caused me quite a bit of confusion as I was setting this up!

I've never done any typescript before, so I could have broken everything (there were a lot of errors in vscode, but I /think/ that's down to me not having everything setup right).

When you have the time, please check over this to make sure I didn't break everything - I didn't test any of these changes (no idea how :disappointed:)
Happy to make further changes if you have any issues :smiley: 

Cheers

### CODE REVIEW

File: src/modules/features/ModuleSetupObsidian.ts [hunk 1]
```diff
@@ -139,11 +139,11 @@ export class ModuleSetupObsidian extends AbstractObsidianModule implements IObsi
...
                     newSettingW.additionalSuffixOfDatabaseName = `${"appId" in this.app ? this.app.appId : ""}`;
-                    const setupJustImport = "Just import setting";
-                    const setupAsNew = "Set it up as secondary or subsequent device";
-                    const setupAsMerge = "Secondary device but try keeping local changes";
-                    const setupAgain = "Reconfigure and reconstitute the data";
-                    const setupManually = "Leave everything to me";
+                    const setupJustImport = "Don't sync anything, just apply the settings.";
+                    const setupAsNew = "This is a new client - sync everything from the remote server.";
+                    const setupAsMerge = "This is an existing client - merge existing files with the server.";
+                    const setupAgain = "This is a new server - reset remote server and use local files.";
```
Comment (vrtmrz) [inline src/modules/features/ModuleSetupObsidian.ts:145]: "This is a new server":  Can you make it a bit more nuanced, like ‘this client creates new server data’?
`This is for a new or broken server`? Is it too long?

File: src/modules/features/SettingDialogue/ObsidianLiveSyncSettingTab.ts [hunk 2]
```diff
@@ -922,22 +922,25 @@ Store only the settings. **Caution: This may lead to data corruption**; database
...
+                    .setName("Manual setup").addButton((text)
+                    .setDesc("Not recommended, but useful if you don't have a Setup URI") => {
+                        text.setButtonText("Start").onClick(async () => {
+                            await this.enableMinimalSetup();
+                        });
                 });
                 new Setting(paneEl)
-                    .setName("Enable LiveSync on this device as the setup was completed manually")
+                    .setName("Enable LiveSync")
+                    .setDesc("Only enable this after configuring either of the above two options.")
```
Comment (vrtmrz) [inline src/modules/features/SettingDialogue/ObsidianLiveSyncSettingTab.ts:943]: Please add the case that completed all configuration manually.

File: src/modules/features/SettingDialogue/ObsidianLiveSyncSettingTab.ts [hunk 3]
```diff
@@ -1526,8 +1533,8 @@ However, your report is needed to stabilise this. I appreciate you for your grea
 
             void addPanel(paneEl, "Fetch settings").then((paneEl) => {
                 new Setting(paneEl)
-                    .setName("Fetch tweaks from the remote")
-                    .setDesc("Fetch other necessary settings from already configured remote.")
+                    .setName("Fetch config from server")
```
Comment (vrtmrz) [inline src/modules/features/SettingDialogue/ObsidianLiveSyncSettingTab.ts:1536]: Please keep `remote` or change something good: we can serve some database (or bucket) on a single server. (It used to be Database, before implemented bucket sync) However, as you fixed, this is a bit ambiguous.

File: src/modules/features/SettingDialogue/ObsidianLiveSyncSettingTab.ts
Comment (vrtmrz) [inline src/modules/features/SettingDialogue/ObsidianLiveSyncSettingTab.ts:1]: Please keep `remote` or change something good: we can serve some database (or bucket) on a single server. (It used to be Database, before implemented bucket sync) However, as you fixed, this is a bit ambiguous.

File: src/modules/features/SettingDialogue/settingConstants.ts [hunk 4]
```diff
@@ -105,48 +105,50 @@ export const SettingInformation: Partial<Record<keyof AllSettings, Configuration
     },
     syncOnSave: {
         name: "Sync on Save",
-        desc: "When you save a file, sync automatically",
+        desc: "Forces a sync of the file when it's saved.",
```
Comment (vrtmrz) [inline src/modules/features/SettingDialogue/settingConstants.ts:108]: Sorry for the ambiguity, but I'm glad your PR uncovered it. This simply starts synchronisation when we save the file, so the file that is synchronised is not limited to this file.

File: src/modules/features/SettingDialogue/settingConstants.ts [hunk 5]
```diff
@@ -105,48 +105,50 @@ export const SettingInformation: Partial<Record<keyof AllSettings, Configuration
...
     },
     disableMarkdownAutoMerge: {
-        name: "Always resolve conflicts manually",
-        desc: "If this switch is turned on, a merge dialog will be displayed, even if the sensible-merge is possible automatically. (Turn on to previous behavior)",
+        name: "Always prompt merge conflicts",
+        desc: "Should we prompt you for every single merge, even if we can safely merge automatcially?",
     },
     writeDocumentsIfConflicted: {
-        name: "Always reflect synchronized changes even if the note has a conflict",
+        name: "Sync changes even if the file has a conflict.",
```
Comment (vrtmrz) [inline src/modules/features/SettingDialogue/settingConstants.ts:151]: If the file has a conflict, the wording `sync` is not correct. The original `Synchronised` was totally ambiguous. How about `Apply the latest transferred change even if the file has a conflict.` or `Write the latest transferred change even...`?

File: src/modules/features/SettingDialogue/settingConstants.ts [hunk 6]
```diff
@@ -259,7 +261,7 @@ export const SettingInformation: Partial<Record<keyof AllSettings, Configuration
     },
     useCustomRequestHandler: {
         name: "Use Custom HTTP Handler",
-        desc: "If your Object Storage could not configured accepting CORS, enable this.",
+        desc: "Enable this if your Object Storage doesn't accept CORS",
```
Comment (vrtmrz) [inline src/modules/features/SettingDialogue/settingConstants.ts:264]: How about `Enable this if your Object Storage is not configurable about CORS`?

**Comment by vrtmrz** [comment]
Thanks to your rewrite, I am very aware that the original English text was ambiguous and difficult to understand. It was misleading and indeed deceptive. Your rewrite has made it much clearer.
I have added many things in greedily and would be grateful for good ideas for them!

File: src/modules/features/ModuleSetupObsidian.ts (hunk 1 above)
Comment (Volkor3-16) [reply src/modules/features/ModuleSetupObsidian.ts]: I agree, my wording was a little simple there - I was trying to keep them as short as possible to make it easy to read at a glance, even if it left out some information.
I've changed it to `Initialise new server data - ideal for new or broken servers.`
There's probably a even better way of wording it, but this seems much better for now :)

File: src/modules/features/SettingDialogue/ObsidianLiveSyncSettingTab.ts
Comment (Volkor3-16) [reply src/modules/features/SettingDialogue/ObsidianLiveSyncSettingTab.ts:1]: I've clarified the strings to say "remote server" - I feel like the word remote on its own can be a little bit confusing - the word server is more generic.

File: src/modules/features/SettingDialogue/settingConstants.ts (hunk 4 above)
Comment (Volkor3-16) [reply src/modules/features/SettingDialogue/settingConstants.ts]: changed to `Starts synchronisation when a file is saved.`

File: src/modules/features/SettingDialogue/settingConstants.ts (hunk 5 above)
Comment (Volkor3-16) [reply src/modules/features/SettingDialogue/settingConstants.ts]: Rewrote these again to the following. 
```
name: "Apply Latest Change if Conflicting",
desc: "Enable this option to automatically apply the most recent change to documents even when it conflicts",
```

File: src/modules/features/SettingDialogue/ObsidianLiveSyncSettingTab.ts [hunk 7]
```diff
@@ -922,22 +922,25 @@ Store only the settings. **Caution: This may lead to data corruption**; database
...
                             eventHub.emitEvent(EVENT_REQUEST_OPEN_SETUP_URI);
                         });
                     });
 
-                new Setting(paneEl).setName("Minimal setup").addButton((text) => {
-                    text.setButtonText("Start").onClick(async () => {
-                        await this.enableMinimalSetup();
-                    });
+                new Setting(paneEl)
+                    .setName("Manual setup").addButton((text)
```
Comment (vrtmrz) [inline src/modules/features/SettingDialogue/ObsidianLiveSyncSettingTab.ts:935]: Please fix here. Probably just only the line has been moved during your work!

```
  new Setting(paneEl)
                    .setName("Connect with Setup URI")
                    .setDesc("This is the recommended method to set up Self-hosted LiveSync with a Setup URI.")
                    .addButton((text) => {
                        text.setButtonText("Use").onClick(() => {
```

**Comment by vrtmrz** [comment]
Thanks for polishing it up! I think that it has got very clear!

Now for other minor things, I hope to ask you for two additional works.
- Fix this [minor issue](https://github.com/vrtmrz/obsidian-livesync/pull/543#discussion_r1867428209)
- run `npx run pretty` to prettify.

I am looking forward to merge your awesome contribution!

**Comment by Volkor3-16** [comment]
Done and done!

One thing to note though, it seems like you set it up to use `crlf` - however the entire project is in `lf` formatting, so I've changed the prettierrc file to use that.

Realistically, it doesn't matter for compatibility, except for ancient versions of windows (winxp or earlier)

**Comment by vrtmrz** [comment]
Thank you for all your fine work! I am so happy to have this merged!
The docs will also be updated soon to reflect the title of the settings!

**Pull request merged**